js/mage/adminhtml/wysiwyg/tinymce/setup.js

```diff
@@ -70,6 +70,12 @@ tinyMceWysiwygSetup.prototype =
         var settings = {
             selector: this.selector,
             config: this.config,
+            allow_conditional_comments: true,
```

Comment:
is this line necessary from your tests?

Reply:
I don't know 😅, I have done so many tests with different combinations, and this is the optimal one.
I am glad about the new version of TinyMCE, but truth be told, it sucks as an editor 😂